src/services/AudiusBackend.js

```diff
@@ -2382,17 +2382,18 @@ class AudiusBackend {
 
     try {
       const { data, signature } = await AudiusBackend.signData()
-      await fetch(`${IDENTITY_SERVICE}/score/hcaptcha`, {
+      return fetch(`${IDENTITY_SERVICE}/score/hcaptcha`, {
```

Comment:
if you return here, you may not get the effect of the try catch you intend to. this fn will now not do the work, but instead a promise that whoever calls this will have to await

Reply:
goooood catch